charts/gatekeeper/values.yaml

```diff
@@ -139,6 +139,7 @@ preUninstall:
     runAsNonRoot: true
     runAsUser: 1000
 podAnnotations: {}
+auditPodAnnotations: {}
```

Comment:
this should not be modified

Reply:
reverting back

Reply:
Wouldn't it will make more sense to name it `auditPodAnnotations`? since on high level, we are setting annotations only on audit pod through this variable

Reply:
@sozercan Why should this not be modified? The idea would be that these are annotations that apply only to the audit pod.

Calling this `pubsubAnnotations` implies that there are annotations specific to pubsub that should only be applied to the audit pod. This is pretty transparently a coupling of our Helm chat with the configuration/behavior of a third-party project (Dapr -- the only project I'm aware of that has the notion of pubsub annotations), which is something we avoid on principle.

Whatever the outcome of this discussion, the name of the config value and the helm chart substitution directive should be in alignment (either `auditPodAnnotations` and `HELMSUBST_AUDIT_POD_ANNOTATIONS` or `pubsubAnnotations` and `HELMSUBST_PUBSUB_ANNOTATION`, not a mixture of the two).